source/whats-new.txt

```diff
@@ -45,6 +43,82 @@ What's New in 1.12
      deprecated. You can create a client and connect in one call
      by using the ``mongo.Connect()`` method.
 
+New features of the 1.12 Go driver release include: 
+
+Queryable Encryption
+~~~~~~~~~~~~~~~~~~~~
+
+The release adds production-ready support for
+:ref:`qe-manual-feature-qe` (QE). To use
+the QE feature, you must use MongoDB Server version 7.0 or later and
+``libmongocrypt`` version 1.8.0 or later.
+  
+This release introduces the ``ClientEncryption.CreateEncryptedCollection()``
+method, which automatically creates data encryption keys when
+creating a new encrypted collection. This method adds the ability to
+fetch KMS credentials automatically from Azure, GCP, and AWS
+environments.
+
+Logging Interface
+~~~~~~~~~~~~~~~~~
+
+You can now record detailed information about connection management and
+command execution.
+
+The logging configuration adds a new ``LogSink`` interface that is
+compatible with the `logr.LogSink <https://pkg.go.dev/github.com/go-logr/logr#LogSink>`__ interface.
+You can use existing ``logr`` adapters with the logging interface
+such as :github:`zerologr <go-logr/zerologr>` or :github:`zapr <go-logr/zapr>`.
+    
+You can also implement your own ``LogSink`` adapter. The following
+example shows how to integrate logging with an existing :github:`zerolog
+<rs/zerolog>` logger:
+
+.. code-block:: go
+
+   sink := zerologr.New(&myLogger).GetSink()
+   loggerOptions := options.Logger().SetSink(sink)
+   options.Client().SetLoggerOptions(loggerOptions)
+
+Additional BSON Marshalling Options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This release adds new features to the ``options`` package to specify
+various BSON marshalling and unmarshalling behaviors.
+
+The following example shows how to set new BSON options on your
+``Client``. The options specify the following behaviors:
+
+- The driver falls back to ``json`` struct tags if ``bson`` struct tags
+  are missing.
+- The driver marshals ``nil`` Go map types as empty BSON documents.
+- The driver marshals ``nil`` Go slice types as empty BSON arrays.
+
+.. code-block:: go
+
+   bsonOpts := &options.BSONOptions{
+   	UseJSONStructTags: true,
+   	NilMapAsEmpty:     true,
+   	NilSliceAsEmpty:   true,
+   }
+   options.Client().SetBSONOptions(bsonOpts)
+
+Simplified Write Concern Specification
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This release simplifies the ``WriteConcern`` API. To learn more about
+the changes, see :ref:`Write Concern <golang-writeconcern>`.
```

Comment:
Question:
Is this a breaking change? I wasn't able to determine what changed from the page. If a breaking change, I think it could be helpful to call that out along with details on what those are.

Reply:
No, the change adds a new "simplified" API and deprecates of some of the existing API. However, it doesn't break backward compatibility.

Reply:
@matthewdale Is there a list that maybe we could add to the Deprecation Notice at the top?